rfd/0066-ip-based-validation.md

```diff
@@ -0,0 +1,107 @@
+---
+authors: Przemko Robakowski(przemko.robakowski@goteleport.com)
+state: draft
+---
+
+# RFD 66 - IP-based validation
+
+## What
+
+Additional validation based on client IP for creating and using certificates. User can define which IP addresses can
+create certificates and from where they can be used.
+
+## Why
+
+It provides additional security against leaked credentials, if adversary gets hold of certificate he won't be able to
+use them outside defined set of machines. It also forms part of user identity.
+
+Relevant issue: [#7081](https://github.com/gravitational/teleport/issues/7081)
+
+## Details
+
+### Configuration
+
+New fields will be added to role allow/deny sections:
+
+* `generator_ips` - list of IP addresses or subnets in CIDR notation that can or can't be used to create certificates
```

Comment:
How about `requestor_ips`? We're really talking about who can requeset a cert, not who can generate a cert, right?